styles.css

```diff
@@ -0,0 +1,71 @@
+p {
+    margin: 0;
+}
+
+.credit-card {
+    background-color: #425ecd;
+    font-size: large;
+    height: 240px;
+    width: 360px;
+    border-radius: 10px;
+    margin: 30px 0 0 20px;
+    display: flex;
+    flex-direction: column;
+}
+
+.credit-card-front > div {
+    margin-left: 40px;
+}
+
+.credit-card-front > p {
+    margin-left: 40px;
+}
+
+.credit-card-front > .number{
+    margin-top: 100px;
+    font-size: xx-large;
+}
+
+.credit-card-front > .information-wrapper {
+    display: inline-flex;
+    align-items: center;
+}
+
+.credit-card-front > .information-wrapper > p:first-child {
+    text-transform: uppercase;
+    display: flex;
+    flex-direction: column;
+    margin: 10px 10px 10px 0;
+    width: 50px;
+}
+
+.magnetic-strip{
+    height: 50px;
+    background-color: black;
+    margin: 30px 0;
+}
+
+.credit-card-back > .information-wrapper {
+    height: 35px;
+    width: 50%;
+    display: flex;
+    flex-direction: row;
+    background-color: lightgrey;
+    justify-content: space-between;
+    margin-left: 10px;
+    align-items: center;
+    padding-left: 10px;
+}
+
+.credit-card-back > .information-wrapper > .name {
```

Comment:
You can simplify this to `.credit-card-back .name`

Reply:
done